README.md

```diff
@@ -317,6 +317,53 @@ The full source code is available [here](./examples/catcma.py).
 </details>
 
 
+#### Maximum a Posteriori CMA-ES [Hamano et al. 2024b]
+MAP-CMA is a method that is introduced to interpret the rank-one update in the CMA-ES from the perspective of the natural gradient.
+The rank-one update derived from the natural gradient perspective is extensible, and an additional term, called momentum update, appears in the update of the mean vector.
```

Comment:
It is better to mention that like "The performance of MAP-CMA is not so different from CMA-ES, as the main motivation of MAP-CMA comes from theoretical understanding of CMA-ES".